src/content/app/genome-browser/hooks/useGenomeBrowser.ts

```diff
@@ -175,10 +175,7 @@ const useGenomeBrowser = () => {
 
     trackConfigs &&
       Object.keys(trackConfigs).forEach((key) => {
-        let trackId = key;
-        if (trackId.match('focus')) {
-          trackId = 'focus';
-        }
+        const trackId = key;
```

Comment:
just a minor thing, we dont need to assign `const trackId= key` if we are removing the `if` statement and the `.match`. we could probably write it this way:
`Object.keys(trackConfigs).forEach((trackId) => {` 
or if we want to go with `key` because on line 180 we are using key but then further down we are using trackId. I think to keep the code clean its better to keep it consistent.

Reply:
good catch. changed now. Thanks